modules/mono/glue/GodotSharp/GodotSharp/Core/StringExtensions.cs

```diff
@@ -530,9 +540,9 @@ public static string GetBaseDir(this string instance)
                 }
             }
 
-            int sep = Mathf.Max(rs.RFind("/"), rs.RFind("\\"));
+            int sep = rs.AsSpan().LastIndexOfAny('/', '\\');
 
-            if (sep == -1)
+            if (sep < 0)
```

Comment:
The C++ implementation seems to use [`== -1`](https://github.com/godotengine/godot/blob/9dd6c4dbac70b28e8156255c3a2b78722772b036/core/string/ustring.cpp#L5038) here.